cognite/client/data_classes/capabilities.py

```diff
@@ -49,21 +49,22 @@ class CurrentUserScope(ScopeBase):
     _scope_name = "currentuserScope"
 
 
-@dataclass(frozen=True)
-class DataSetScope(ScopeBase):
-    _scope_name = "datasetScope"
-    data_set_ids: list[int]
-
-
 @dataclass(frozen=True)
 class IDScope(ScopeBase):
     _scope_name = "idScope"
     ids: list[int]
 
 
 @dataclass(frozen=True)
-class ExtractionPipelineScope(IDScope):
+class ExtractionPipelineScope(ScopeBase):
```

Comment:
I changed this one to avoid having a inheritance hierarchy deeper than one, as the `ScopeBase` should be the interface of al scope implementations. I also think that `ExtractionPipelineScope` is not a type of `IDScope`, thus it would be misleading to have it as a parent.